products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
I've worked around or punted on the parameterized image files (at least for now).  RHEL 6 users will get some rhel7 screenshots which won't cause a big deal.  Looks like everything but blinkr passed.  

I left the awestruct paramters populating some AsciiDoctor attributes.  That doesn't seem to cause a problem,  It was the hack/workaround in the CDN image links that  didn't work. (though they don't fail for a local docker environment, which is somewhat understandable.)